docs/status.md

```diff
@@ -44,8 +44,8 @@ one of the following tracking labels.
 | batch_norm_inference     | no            | yes*         | yes*           | no              | no          |
```

Comment:
Is there a plan to expand this work to include other ops with I64ElementsAttr attributes, e.g. `slice`?